sysbench_plugin.py

```diff
@@ -4,205 +4,414 @@
-                                         "name": "Number of events", "description": "Maximum number of events"})
-    time: typing.Optional[int] = field(default=10, metadata={
-                                       "name": "Time", "description": "Limit for total execution time in seconds"})
-    memoryblocksize: typing.Optional[str] = field(default='1KiB', metadata={
-                                                  "name": "Block Size", "description": "size of memory block for test in KiB/MiB/GiB"})
-    memorytotalsize: typing.Optional[str] = field(default='100G', metadata={
-                                                  "name": "Total Size", "description": "Total size of data to transfer in GiB"})
-    memoryscope: typing.Optional[str] = field(default='global', metadata={
-                                              "name": "Memory Scope", "description": "Memory Access Scope(global/local)"})
-    memoryoperation: typing.Optional[str] = field(default='write', metadata={
-                                                  "name": "Memory Operation", "description": "Type of memory operation(write/read)"})
+        metadata={
+            "name": "Operation",
+            "description": "Sysbench Operation to perform",
+        }
+    )
+    threads: typing.Optional[int] = field(
+        default=1,
+        metadata={
+            "name": "Threads",
+            "description": "Number of worker threads to create",
+        },
+    )
+    events: typing.Optional[int] = field(
+        default=0,
+        metadata={
+            "name": "Number of events",
+            "description": "Maximum number of events",
+        },
+    )
+    time: typing.Optional[int] = field(
+        default=10,
+        metadata={
+            "name": "Time",
+            "description": "Limit for total execution time in seconds",
+        },
+    )
+    memoryblocksize: typing.Optional[str] = field(
+        default="1KiB",
+        metadata={
+            "name": "Block Size",
+            "description": "size of memory block for test in KiB/MiB/GiB",
+        },
+    )
+    memorytotalsize: typing.Optional[str] = field(
+        default="100G",
+        metadata={
+            "name": "Total Size",
+            "description": "Total size of data to transfer in GiB",
+        },
+    )
+    memoryscope: typing.Optional[str] = field(
+        default="global",
+        metadata={
+            "name": "Memory Scope",
+            "description": "Memory Access Scope(global/local)",
+        },
+    )
+    memoryoperation: typing.Optional[str] = field(
+        default="write",
+        metadata={
+            "name": "Memory Operation",
+            "description": "Type of memory operation(write/read)",
+        },
+    )
 
 
 @dataclass
 class LatencyAggregates:
     avg: float = field(
-        metadata={"name": "Average", "description": "Average Latency"})
+        metadata={"name": "Average", "description": "Average Latency"}
```

Comment:
There are some annoying inconsistencies in the schema between using in-line or line-returned formatting. I'd prefer they were all formatted as line-returned.